src/rpcserver/.clang-format

```diff
@@ -0,0 +1,66 @@
+# Generated from CLion C/C++ Code Style settings
```

Comment:
add `clang-format` / `clang-tidy` to CI to make sure this is indeed used (using cmake?)